ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OGridSearch.scala

```diff
@@ -202,16 +205,18 @@ class H2OGridSearch(override val uid: String)
   def getGridModelsParams(): DataFrame = {
     ensureGridSearchIsFitted()
     val hyperParamNames = getHyperParameters().keySet().asScala.toSeq
+    val h2oToSwParamMap = getAlgo().getSWtoH2OParamNameMap().map(_.swap)
     val rowValues = gridModels.zip(gridModels.map(_.uid)).map {
       case (model, id) =>
-        val outputParams = model.getTrainingParams().filter { case (key, _) => hyperParamNames.contains("_" + key) }
+        val outputParams =
+          model.getTrainingParams().filter { case (key, _) => hyperParamNames.contains(h2oToSwParamMap(key)) }
```

Comment:
Isn't this a breaking change? it seems that `hyperParamNames` contains SW names only.

Reply:
Do you mean that in case of `outputParams` we always output SW names? That is a good catch, I will try to make sure that the param names remain consistent in the fix release

Reply:
In this statement `hyperParamNames.contains(h2oToSwParamMap(key))`, the `key` is always converted to sw name. What if `hyperParameNames` contains old h2o names? This comment is als related to [the line 218](hyperParamNames.contains(h2oToSwParamMap(key))).

Reply:
Thanks, will fix

Reply:
I decided to deprecated the behaviour in 3.30 and in 3.32 support just the new version. I think this solution will be cleaner than supporting two paths in the code